build.sbt

```diff
@@ -71,16 +71,15 @@ lazy val commonSettings = Seq(
   )
 )
 
-lazy val root = crossProject(JSPlatform, JVMPlatform)
+lazy val krop = crossProject(JSPlatform, JVMPlatform)
   .in(file("."))
   .settings(moduleName := "krop")
-lazy val rootJs =
-  root.js.aggregate(
-    core.js
-  )
+
+lazy val kropJs =
```

Comment:
Renamed this so it is easy to identify in IDEs when there are many recently opened projects. `root` is hard to recognize.